### PR TITLE
Improve password storage

### PR DESCRIPTION
**Pull request by HansVanpee**
The storage collection is selected in a desktop agnostic way, relying on libsecret functionality only.

### CODE REVIEW

File: src/common/pwstorage/backend_libsecret.c [hunk 1]
```diff
@@ -30,7 +30,6 @@
 #include <json-glib/json-glib.h>
 
 #define DARKTABLE_KEYRING PACKAGE_NAME
-
```
Comment (HansVanpee) [inline src/common/pwstorage/backend_libsecret.c]: Don't rely on Gnome Keyring's "Login" collection anymore.

**Comment by LebedevRI** [comment]
@HansVanpee as it was (IIRC) said, without some documentation saying otherwise, our current approach, now that it actually works, is no better or worse than this. (meaning: if no docs say that this is the right way, we should probably keep what we already have)
Also, why there is 2 PR's? #1214 #1215

**Comment by HansVanpee** [comment]
I think it works perfectly fine now (for my use case: Gnome
Shell/libsecret): by default passwords go to the "login" collection just
like with any other software package. It is completely transparent to the
user and storage is persistent.
The second PR came after some exchanges with Tobias Ellinghaus about this
matter. Both can be discarded now.

2016-07-23 21:09 GMT+02:00 Roman Lebedev <notifications@github.com>:

**Comment by TurboGit** [comment]
Same for me, all is fine with current master version.

**Comment by LebedevRI** [comment]
In any case, this PR is bogus - no changes, originates from master branch, and has lot's of "Merge remote-tracking branch 'upstream/master'"
For master branch, one should use `git rebase upstream/master master`, not `git merge`

**Pull request closed without merging**